app/models/profile.rb

```diff
@@ -217,6 +217,11 @@ def irs_attempts_api_tracker
 
   private
 
+  def confirm_that_profile_can_be_activated!
+    reason = reason_not_to_activate
+    raise reason if reason
```

Comment:
unsure if it reads any better to others, but seeing the same variable repeated a bunch like this had me do a double take:
```suggestion
    if (reason = reason_not_to_activate)
      raise reason
    end
```

and Rubocop might yell at you and force this:
```suggestion
    raise reason if (reason = reason_not_to_activate)
```

Reply:
Discussed this suggestion back-channel with @zachmargolis , and other options are `raise reason_not_to_activate if reason_not_to_activate`, or write a little method `raise_if_not_nil(reason_not_to_activate).`

Reply:
... i had initially refactored `raise reason_not_to_activate if reason_not_to_activate` to the current `raise reason if reason` in favor of not calling the method twice; however, it looks like the readability trade off isn't worth it. Thanks for the feedback 👍🏿 
 will go back to `raise reason_not_to_activate if reason_not_to_activate`